### ISSUE TITLE
[Feature] Isometric charts

### ISSUE THREAD

**Issue by dapeleg-dn** [apache/echarts#18898]
### What problem does this feature solve?

Having isometric charts in your vast and fantastic collection of charts will be nice. 

One use case I see is to describe network topology. 

Given the excellent integration with Grafana using the [Echarts Panel made by Volkov Labs](https://volkovlabs.io/plugins/volkovlabs-echarts-panel/), it will be beneficial to visualize real-time network metrics over the visualized network topology. Things like the rate of network traffic, allowed vs. blocked traffic, firewall visualization, and many other metrics can be nicely visualized using colors, line widths, and animations. 

Another use case is the visualization of Solar power systems like [Victron Cerbo GX](https://youtu.be/1Gu9Ezugojc?t=544).

### What does the proposed API look like?

Basically, same as the other network charts. Maybe need to add properties for the location of the objects in the space and the user's perspective.

You can get inspiration from [yWorks Isometric diagrams](https://www.yworks.com/pages/isometric-graph-drawing).

**Comment by helgasoft** [comment]
These are interesting charts, the [demo](https://live.yworks.com/demos/showcase/isometricdrawing) even has live editing. But there are no data(metrics) associated with the nodes, thus your Feature request.
Probably related to #18943, see [image](https://i.stack.imgur.com/UdMWy.png).
